### PR TITLE
[minor_change] Adding missing attributes for fabric wide metrics

### PR DESCRIPTION
**Pull request by akinross**
solves #923

### CODE REVIEW

File: website/docs/d/fabric_wide_settings.html.markdown [hunk 1]
```diff
@@ -32,11 +32,17 @@ data "aci_fabric_wide_settings" "example" {}
 * `description` - (Optional) Description of object Fabric-Wide Settings Policy.
 * `name_alias` - (Optional) Name alias of object Fabric-Wide Settings Policy.
 * `disable_ep_dampening` - (Optional) Disable Ep Dampening knob of object Fabric-Wide Settings Policy. 
+* `domain_validation` - (Optional) Validate that static path is added but no domain is associated to an EPG.
```
Comment (lhercot) [inline website/docs/d/fabric_wide_settings.html.markdown:35]: Validate that the correct physical domain is added before associating a new static path to an EPG.

File: website/docs/r/fabric_wide_settings.html.markdown [hunk 2]
```diff
@@ -46,11 +52,17 @@ Users can use the resource of type `aci_fabric_wide_settings` to change the conf
 * `description` - (Optional) Description of object Fabric-Wide Settings Policy.
 * `name_alias` - (Optional) Name alias of object Fabric-Wide Settings Policy.
 * `disable_ep_dampening` - (Optional) Disable Ep Dampening knob of object Fabric-Wide Settings Policy. Allowed values are "yes" and "no".
+* `domain_validation` - (Optional) Validate that static path is added but no domain is associated to an EPG.
```
Comment (lhercot) [inline website/docs/r/fabric_wide_settings.html.markdown:55]: Validate that the correct physical domain is added before associating a new static path to an EPG.

**Review by lhercot — CHANGES_REQUESTED** [review]

File: website/docs/d/fabric_wide_settings.html.markdown (hunk 1 above)
Comment (akinross) [reply website/docs/d/fabric_wide_settings.html.markdown]: ok

File: website/docs/r/fabric_wide_settings.html.markdown (hunk 2 above)
Comment (akinross) [reply website/docs/r/fabric_wide_settings.html.markdown]: ok

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**